packages/mds-transactions-service/repository/index.ts

```diff
@@ -190,6 +235,57 @@ class TransactionReadWriteRepository extends ReadWriteRepository {
     }
   }
 
+  /**
+   * @deprecated
+   * **WARNING: This should ONLY be used during tests! Hence adding the deprecated flag.**
+   * Deletes all transactions from the DB.
+   */
+  public deleteAllTransactions = async () => {
+    const { connect } = this
+    try {
+      const connection = await connect('rw')
+      const repository = await connection.getRepository(TransactionEntity)
+
+      await repository.query(`DELETE FROM ${repository.metadata.tableName};`)
+    } catch (error) {
+      throw RepositoryError(error)
+    }
+  }
+
+  /**
+   * @deprecated
+   * **WARNING: This should ONLY be used during tests! Hence adding the deprecated flag.**
+   * Deletes all transaction operations from the DB.
+   */
+  public deleteAllTransactionOperations = async () => {
+    const { connect } = this
+    try {
+      const connection = await connect('rw')
+      const repository = await connection.getRepository(TransactionOperationEntity)
+
+      await repository.query(`DELETE FROM ${repository.metadata.tableName};`)
+    } catch (error) {
+      throw RepositoryError(error)
+    }
+  }
+
+  /**
+   * @deprecated
+   * **WARNING: This should ONLY be used during tests! Hence adding the deprecated flag.**
+   * Deletes all transaction statuses from the DB.
+   */
+  public deleteAllTransactionStatuses = async () => {
+    const { connect } = this
+    try {
+      const connection = await connect('rw')
+      const repository = await connection.getRepository(TransactionStatusEntity)
+
+      await repository.query(`DELETE FROM ${repository.metadata.tableName};`)
+    } catch (error) {
+      throw RepositoryError(error)
+    }
+  }
+
```

Comment:
I added these because test isolation (even within a given file 🙀 ) is super nice, and makes the tests *vastly* more readable and maintainable. No side effects for you *smacks with a stick*!

Reply:
Might be a good KTLO project to add something similar to the template. I've been writing my tests in such a way that I don't need to dump the tables in between each group of tests, but this would be nice to have.

Reply:
Another suggestion... put this at the beginning of each of these methods....
```ts
if (process.env.NODE_ENV !== 'test') {
   throw new Error(`This method is only supported when executing tests`)
}
```
It seems like it might be possible to add a generic method to the base `ReadWriteRepository` class in @mds-repository that took an entity as a parameter and did this. So you could just call `deleteAll(TransactionStatusEntity)` or something like that.

Finally, this command would return the table to an even more fresh condition:
```sql
TRUNCATE ${repository.metadata.tableName} RESTART IDENTITY;
```

Reply:
Good idea for the safeguard @drtyh2o and +1 to the KTLO idea @janedotx. I'm going to add the safeguard for now, and will look to moving this to the base class in a subsequent PR